tests/core/test_expectation_suite.py

```diff
@@ -603,3 +604,55 @@ def test_get_expectations_by_domain_type(
         exp4,
         column_pair_expectation,
     ]
+
+
+class DataContextSendUsageMessageSpy:
+    def __init__(self):
+        self.messages = []
+
+    def send_usage_message(
+        self,
+        event,
+        event_payload,
+        success,
+    ):
+        self.messages.append(
+            {
+                "event": event,
+                "event_payload": event_payload,
+                "success": success,
+            }
+        )
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "success",
+    [
+        pytest.param(True, id="success=True"),
+        pytest.param(False, id="success=False"),
+    ]
+)
+def test_expectation_suite_send_usage_message(success: bool):
+    """Ensure usage stats event is sent on expectation suite."""
+
+    dc_message_spy = DataContextSendUsageMessageSpy()
+
+    suite = ExpectationSuite(
+        expectation_suite_name="warning",
+        expectations=[],
+        meta={"notes": "This is an expectation suite."},
+        data_context=dc_message_spy,
+    )
+
+    suite.send_usage_event(success=success)
+
+    assert dc_message_spy.messages
+    assert len(dc_message_spy.messages) == 1
+    assert dc_message_spy.messages[0] == {
+        "event": UsageStatsEvents.EXPECTATION_SUITE_ADD_EXPECTATION.value,
```

Comment:
This isn't blocking, just a question. Do we want other tests/parameterizations that send in event and event_payload also? 
You could parameterize the inputs as a `parameterized_kwargs` dict and then call `suite.send_usage_event(**parameterized_kwargs)`. When doing the assert you'd need to check for presence/absence of a key in `parameterized_kwargs` to know whether to use the default value or the parameterized value. If that's ugly, we could have a separate test that uses this spy where you send it a nondefault values for these args.

Reply:
Good call - when we test `context.send_usage_message()` I think that makes sense. For this method though `suite.send_usage_event(success: bool)` only the `success` parameter is available, the rest are set within the method call.